### PR TITLE
Added custom scrollbar to sidenav

### PR DESCRIPTION
**Pull request by FireQueen-3010**
image

### CODE REVIEW

**Comment by FireQueen-3010** [comment]
Please review @LordShenron @ArunTeltia :)

File: src/components/PersistentDrawerLeft/PersistentDrawerLeft.css [hunk 1]
```diff
@@ -1,6 +1,39 @@
...
+									   color-stop(0.78, #B3D4FC),
+									   color-stop(0.86, #00CCFF));
+   
+   /* background-image:-webkit-linear-gradient(20deg,#00CCFF 20%,transparent 20%,transparent 40%,
+   #00CCFF 40%,#00CCFF 60%,transparent 60%,transparent 80%,
+   #00CCFF 80%);                   */
 }
+
+/*for firefox*/
+.MuiDrawer-paper{
```
Comment (ArunTeltia) [inline src/components/PersistentDrawerLeft/PersistentDrawerLeft.css:30]: great job figuring out for mozilla

File: src/components/PersistentDrawerLeft/PersistentDrawerLeft.css [hunk 2]
```diff
@@ -1,6 +1,39 @@
...
+  background: rgb(0,0,0);
+  background-clip: content-box; 
+}
+
+.MuiDrawer-paper::-webkit-scrollbar-thumb {
+  border-radius: 10px;
+  background-color: #B3D4FC;
+  background-image: -webkit-gradient(linear,
+									   left bottom,
+									   left top,
```
Comment (ArunTeltia) [inline src/components/PersistentDrawerLeft/PersistentDrawerLeft.css:19]: looks clean just the indentation problem
Can be merge tho will resolve this issue after i add eslint stylelint husky and storybook

**Review by ArunTeltia — APPROVED** [review]

**Comment by ArunTeltia** [comment]
Merge it @LordShenron

File: src/components/PersistentDrawerLeft/PersistentDrawerLeft.css (hunk 1 above)
Comment (FireQueen-3010) [reply src/components/PersistentDrawerLeft/PersistentDrawerLeft.css:30]: Thank you :)

File: src/components/PersistentDrawerLeft/PersistentDrawerLeft.css (hunk 2 above)
Comment (FireQueen-3010) [reply src/components/PersistentDrawerLeft/PersistentDrawerLeft.css:19]: should I fix the indentation?

Comment (ArunTeltia) [reply src/components/PersistentDrawerLeft/PersistentDrawerLeft.css:19]: Yes fix it

Comment (FireQueen-3010) [reply src/components/PersistentDrawerLeft/PersistentDrawerLeft.css]: Done :) @ArunTeltia

**Pull request merged**